### PR TITLE
Implements player traits methods

### PR DESCRIPTION
**Pull request by JJendryka**
Changes Player method `add_to_trait` to `change_traits` which now works with dictionaries of changes. Implements `set_traits` (also dictionaries). Adds `has_traits` method and implements it

### CODE REVIEW

**Comment by JJendryka** [comment]
[Implement traits related methods in Player class](https://trello.com/c/3XMQZngK/28-implement-traits-related-methods-in-player-class)

File: src/player.py [hunk 1]
```diff
@@ -19,13 +20,22 @@ def get_state(self):
...
-        """Set the trait to 'value'"""
-        pass
+    def set_traits(self, traits):
+        """Set all traits in `traits` (dictionary name to value)"""
+        for trait, value in traits.items():
+            self.traits[trait] = value
 
-    def add_to_trait(self, name, value):
+    def change_traits(self, traits):
         """Add 'value' to 'trait'"""
```
Comment (Jackojc) [inline src/player.py]: Change function name to `update_traits`

File: src/player.py [hunk 2]
```diff
@@ -4,6 +4,7 @@ class Player(object):
     """Describe the player, his inventory, traits, position and history"""
 
     def __init__(self, state):
+        self.traits = {}
```
Comment (Jackojc) [inline src/player.py]: add self.state and state = None

**Comment by Hazurl** [comment]
Ok I will stop to resolve conflicts... Again this weird problem

**Pull request merged**